geometry_msgs/Point centroid
string label
uint32 size
